src/test/java/util/CsvReaderTest.java

```diff
@@ -0,0 +1,30 @@
+package util;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class CsvReaderTest {
+
+    @Test
+    void 파일을_정상적으로_가져온다() {
+        List<String[]> parsedResult = CsvReader.readFile("src/main/resources/attendances.csv");
+
+        Map<String, String> attendances = new HashMap<>();
+        for (int i = 0; i < 4; i++) {
+            attendances.put(parsedResult.get(i)[0], parsedResult.get(i)[1]);
+        }
+        Map<String, String> expectedResult = new HashMap<>();
+        expectedResult.put("쿠키", "2024-12-13 10:08");
+        expectedResult.put("빙봉", "2024-12-13 10:07");
+        expectedResult.put("빙티", "2024-12-13 10:07");
+        expectedResult.put("이든", "2024-12-13 10:07");
```

Comment:
```suggestion
        Map.of(
            "쿠키", "2024-12-13 10:08",
            "빙봉", "2024-12-13 10:07",
            "빙티", "2024-12-13 10:07",
            "이든", "2024-12-13 10:07");
```
이 부분 위의 방식을 활용해보면 어떨까요?
두 방식의 차이를 알아보는 것도 좋겠네요!
다른 컬렉션들도 Java 버전에 따라 어떤 메서드가 있는지 알아보면 유용하게 쓰일 듯 해요.

Reply:
오! new HashMap<>() 방식은 변경 가능하고 Map.of 방식은 정적 팩토리 메서드로 변경 불가능합니다!